website/app/templates/about.hbs

```diff
@@ -2,7 +2,24 @@
 
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
+    
+    <h2 class="doc-text-h2">Principles</h2>
+    <p class="doc-text-body">It's important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our consumers by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p> 
+
+    <h2 class="doc-text-h2">Accessibility statement</h2>
+    <p class="doc-text-body">Accessibility is not just a requirement, it is a part of our company culture, grounded in <a href="https://www.hashicorp.com/our-principles" rel="noopener noreferrer" target="_blank">our company principles</a> such as integrity, communication and kindness. Read more about our intentions for an accessible design system in our <a href="/about/accessibility-statement/" target="_blank" rel="noopener noreferrer">accessibility statement</a>.</p>
+    
+    <h2 class="doc-text-h2">Getting started for designers</h2>
+    <p class="doc-text-body">Learn how to use our components and other resources by reading our guide to <a href="/getting-started/for-designers/" target="_blank" rel="noopener noreferrer">getting started for designers</a>.</p>
+    
+    <h2 class="doc-text-h2">Getting started for engineers</h2>
+    <p class="doc-text-body">While the design system components are conveniently packaged in a standard Ember addon, we still recommend reading through our guide to <a href="/getting-started/for-engineers/" target="_blank" rel="noopener noreferrer">getting started for engineers</a> to better understand some specific conventions we have put in place to help engineers focus on what is really important in their application, instead of thinking about component APIs.</p>
+    
+    <h2 class="doc-text-h2">Contributions</h2>
+    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute-- and there are quite a few ways our consumers are encouraged to participate! Read our <a href="/getting-started/contribution/" target="_blank" rel="noopener noreferrer">contribution</a> documentation to find the different ways anyone can contribute to Helios.</p>
```

Comment:
Use em dash, & remove space:

"... how people can contribute-- and ..."
=>
"... how people can contribute—and ..."